src/Application/Application.php

```diff
@@ -80,40 +83,32 @@ public function run(array $inputArgs): int
     /**
      * @param Script[] $scripts
      */
-    public function showListing(array $scripts): void
+    private function showListing(array $scripts): void
     {
         $this->cliMate->green()->bold('Available commands:')->br();
 
         if (!count($scripts)) {
-            $this->cliMate->yellow()->bold('-> Currently no scripts available');
+            $this->cliMate->yellow()->bold("-> Currently no scripts available\n");
+
+            return;
         }
 
         $paddingSize = $this->getPaddingSize($scripts);
         $padding = $this->cliMate->padding($paddingSize)->char(' ');
 
-        $scriptEnvironment = false;
-
+        $scriptEnvironment = 'default';
```

Comment:
Wouldn't it make sense to introduce a const for this?

Reply:
TBH it doesn't really matter what the initial value is. Only this way infection-php will not complain that it can change the value to anything else